chocolatey/datahelix/tools/chocolateyinstall.ps1

```diff
@@ -0,0 +1,25 @@
+﻿$ErrorActionPreference = 'Stop';
+$toolsDir   = "$(Split-Path -parent $MyInvocation.MyCommand.Definition)"
+$version    = $env:ChocolateyPackageVersion
+$url32      = "https://github.com/finos/datahelix/releases/download/v$($version)/datahelix.zip"
+$url64      = $url32
+$checksum32 = ''
+$checksum64 = ''
+
+$packageArgs = @{
+  packageName   = $env:ChocolateyPackageName
+  unzipLocation = $toolsDir
+  url           = $url32
+  url64         = $url64
+
+  softwareName  = 'datahelix*'
+  checksum      = $checksum32
+  checksumType  = 'sha256'
+  checksum64    = $checksum64
+  checksumType64= 'sha256'
+}
+
+Write-Host "Installing Data Helix: $($version)"
+Install-ChocolateyZipPackage @packageArgs
+
+Install-BinFile -name "datahelix" -path "$env:ChocolateyInstall\datahelix\bin\datahelix.bat"
```

Comment:
does this update the `%PATH%` environment variable?
if not the powershell command is:
`[System.Environment]::SetEnvironmentVariable('JAVA_HOME','C:\apps\jdk-8.0.232.09',[System.EnvironmentVariableTarget]::User)`

Reply:
It is supposed to instruct chocolatey to create a 'shim'